pub/go.mod

```diff
@@ -2,9 +2,10 @@ module github.com/okteto/pipeline-with-terraform/pub
 
 go 1.17
```

Comment:
@odedlaz could you also update the go version in both go.mod files?

Reply:
@rberrelleza , done.